Foundation/NSCharacterSet.swift

```diff
@@ -168,6 +168,10 @@ public class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
         return CFCharacterSetCreateCopy(kCFAllocatorSystemDefault, self._cfObject)
     }
     
+    public override func mutableCopy() -> AnyObject {
```

Comment:
It looks like the problem is bigger indeed: the problem with `_NSObjectRepresentableBridge` is not only `Int`, none of the objects, that adopt `_NSObjectRepresentable` return `object` for `object as? _NSObjectRepresentable` statement.

I faced with it when I was trying to fix https://bugs.swift.org/browse/SR-251 . Tests are not passing because of this issue with protocols.